### DISCUSSION TITLE
Does not open metamask app to sign when trying to connect from website (opens app-store)

### DISCUSSION THREAD

**Discussion by firuzinho** [WalletConnect/walletconnect-monorepo#1618] [Developer Support]
Hello, I would like to clarify, in version 1.0 there was a small bug in which, when authorizing through a metamask, the application opened the app store or a metamask once, and the second time (when you need to sign a token) it opened the app store, not a metamask. Is this fixed in version 2.0 or is this a bug not on your part? Thanks.

**Comment by xzilja** [comment]
v2 should have much better deep-linking support, so hopefully it is not an issue anymore. Also you can test this out by visiting https://web3modal-dev.pages.dev/ on your mobile device

**Reply by dnmintroad** [reply]
Hi @IljaDaderko , are you able to share the code example used for https://web3modal-dev.pages.dev/?

**Reply by xzilja** [reply]
Sure thing, it is our react example
https://github.com/WalletConnect/web3modal/tree/V2/examples/react

**Reply by xzilja** [reply]
You can find latest stuff on `beta.8` branch
